exoscale/provider.go

```diff
@@ -4,19 +4,26 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"net/http"
 	"os"
 	"os/user"
 	"path/filepath"
 	"strings"
 	"time"
 
-	ini "gopkg.in/ini.v1"
-
 	"github.com/exoscale/egoscale"
 	exo "github.com/exoscale/egoscale/v2"
+	exov2 "github.com/exoscale/egoscale/v2"
```

Comment:
I like this. We should call it `exov2` in all repos where v1 is still in use.
Line 15 can probably be deleted now?